dev-client/src/screens/SlopeScreen/utils/renderValues.ts

```diff
@@ -23,13 +23,38 @@ export const renderSteepness = (
   {slopeSteepnessDegree, slopeSteepnessPercent, slopeSteepnessSelect}: SoilData,
 ) =>
   slopeSteepnessSelect
-    ? t(`slope.steepness.select_labels.${slopeSteepnessSelect}`)
+    ? renderSlopeSteepnessSelectInline(t, slopeSteepnessSelect)
     : typeof slopeSteepnessPercent === 'number'
       ? `${slopeSteepnessPercent.toFixed(0)}%`
       : typeof slopeSteepnessDegree === 'number'
         ? `${slopeSteepnessDegree}°`
         : undefined;
 
+export const renderSteepnessForNarrowDisplay = (
+  t: TFunction,
+  {slopeSteepnessDegree, slopeSteepnessPercent, slopeSteepnessSelect}: SoilData,
+) =>
+  slopeSteepnessSelect
+    ? t(`slope.steepness.select_labels.${slopeSteepnessSelect}`) +
+      '\n' +
+      t(`slope.steepness.select_labels.${slopeSteepnessSelect}_PERCENT`)
+    : typeof slopeSteepnessPercent === 'number'
+      ? `${slopeSteepnessPercent.toFixed(0)}%`
+      : typeof slopeSteepnessDegree === 'number'
+        ? `${slopeSteepnessDegree}°`
+        : undefined;
```

Comment:
question: why are we using a bare newline?
issue: the percentage and degree strings should be localized.
issue: multi-level nested ternaries are too hard to read/follow. please pull this code into a function.

Reply:
1. There are multiple places that we display the slope data. Most places display it with a space between the name and percentage range (1st picture). But in the new screen we want to display it with the name on one line and the percentage range on a second line (2nd picture).
Is there a preferred way to do that?
    1st picture: ![Screenshot 2024-05-21 at 4 02 33 PM](https://github.com/techmatters/terraso-mobile-client/assets/5590815/c2ed0371-5e50-4936-a61c-3dd5bef716cb)
    2nd picture: ![Screenshot 2024-05-21 at 4 02 49 PM](https://github.com/techmatters/terraso-mobile-client/assets/5590815/c8efacd2-e762-45d6-ae76-37f144efe4c7)

2. Ok, I will plan to add `"percentage": "{{number}}%"` and `"degree": "{{number}}°"` to `en.json`

3. Not sure which code you are suggesting extracting, but I'll plan to just use if / else if / else here for readability.

Reply:
Here are two ways to avoid the newline in the code:
• control the wrapping via the container/contain width
• use separate `<Text>` components for the steepness description and steepness percent

Reply:
FYI, we already have a standard means of doing locale-friendly percentage formatting in `util.ts:formatPercent`.

Reply:
@paulschreiber Would it be fine to add `"{{steepness_name}}\n{{steepness_percent}}"` to en.json (since there are many other strings with newlines in there)?

Reply:
It'd be better not to do that. I believe the other places where we have newlines are for static text with paragraph breaks. These are two distinct entities.